lib/metrics.py

```diff
@@ -57,6 +59,23 @@ def emit(self, stats):
             self.fallback_emitter.emit(stats)
 
 
+class LoggingHeartbeatEmitterThread(threading.Thread):
+    def __init__(self, interval):
+        super(LoggingHeartbeatEmitterThread, self).__init__()
+        self.interval = interval
+
+    def run(self):
+        logger.debug(
+            'Starting metrics emitter with interval %d' % self.interval
```

Comment:
For logging, the following syntax is best:

`logger.debug('Foo %s, bar)`, i.e. with the arguments to be merged in as arguments to logger.debug. (see: https://docs.python.org/3/library/logging.html#logging.Logger.debug)

This is better than plain string interpolation (or string concatenation, or `string.format`), as the arguments are only evaluated if the message is actually logged. (i.e. if your logging level is set higher, the string will not be constructed).